docs/AdminResource.md

```diff
@@ -281,6 +284,30 @@ const App = () => (
 export default App;
 ```
 
+### `authentication`
+
+The `authentication` props expect an object with `loginClient`, `logoutClient`, and `checkCredentials` methods, to control the application authentication strategy:
+
+```js
+const authentication = {
+    loginClient(username, password) { /* ... */ },
```

Comment:
That's because it's a client to an auth API. 

But your remark made me reconsider the `authentication` signature: instead of `loginClient` and `logoutClient`, it should expect an `authClient` - similar  to `restClient`. I'll refactor the code accordingly.